docs/network-flow-visibility.md

```diff
@@ -1,12 +1,81 @@
+
+For visualizing the network flows, Antrea monitors the flows in conntrack module. These flows are converted to flow records
+and are sent to the configured flow controller. High-level design is given below:
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow_visibility.png" width="600" alt="Network Flow Visibilty">
+
+## Flow Exporter feature
+
+In Antrea, the basic building block for the Network Flow Visibility is the **Flow Exporter feature**. Flow Exporter operates
+at Antrea Agent; it builds and maintains a connection store by polling and dumping flows from conntrack module periodically.
+The connections from connection store are exported to a flow collector using IPFIX protocol, and for this purpose we
+use [go-ipfix](https://github.com/vmware/go-ipfix) library.
+ 
+### Configuration
+
+To enable the Flow Exporter feature at the Antrea Agent, the following config parameters have to be set in the Antrea Agent ConfigMap as shown 
+below. We provide some examples for the parameter values.
+
+```yaml
+  antrea-agent.conf: |
+    # FeatureGates is a map of feature names to bools that enable or disable experimental features.
+    featureGates:
+    # Enable flowexporter which exports polled conntrack connections as IPFIX flow records from each agent to a configured collector.
+      FlowExporter: true
+
+    # Provide flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp. This also enables
+    # the flow exporter that sends IPFIX flow records of conntrack flows on OVS bridge. If no L4 transport proto is given,
+    # we consider tcp as default.
+    flowCollectorAddr: "192.168.86.86:4739:tcp"
+
+    # Provide flow poll interval as a duration string. This determines how often the flow exporter dumps connections from the conntrack module.
+    # Flow poll interval should be greater than or equal to 1s (one second).
+    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+    flowPollInterval: "1s"
+
+    # Provide flow export frequency, which is the number of poll cycles elapsed before flow exporter exports flow records to
+    # the flow collector.
+    # Flow export frequency should be greater than or equal to 1.
+    flowExportFrequency: 5
+``` 
+
+Please note that if the flowPollInterval and flowExportFrequency parameters are not set with the Flow Exporter feature enabled,
+then they take the default values of 5s and 12 respectively. However, the flowCollectorAddr parameter must be provided for the Flow Exporter
+feature to work. 
+
+### Supported capabilities
+
+Currently, the Flow Exporter feature provides visibility to Pod-to-Pod, Pod-to-Node, Node-to-Pod, Node-to-Node and Pod-to-Service
+network flows along with the associated statistics such as byte throughput, packet throughput, cumulative byte count, cumulative packet
+count etc. Pod-To-Service flow visibility is supported only when Antrea Proxy enabled. In the future, this support will
```

Comment:
I am not sure if we want to support kube-proxy for long. Consider removing the last sentence.

Reply:
Sure, Jianjun removed it. A follow-up question- Even though kube proxy is not going to be the default, will there be no option exposed to turn it on?

Reply:
I think we have not concluded this yet. But I personally hope Antrea Proxy will be the default and probably for the majority of clusters.